case-lib/pipeline.sh

```diff
@@ -1,7 +1,27 @@
 #!/bin/bash
 
+# This function will evaluate pipeline parameter related shell code generated by
```

Comment:
Very useful comment considering this function should be called `something_IMPORT()` (import from Python script). Don't change the name in this PR.